CHANGELOG.rst

```diff
@@ -1,5 +1,40 @@
 .. _changelog:
 
+0.31.0
+------
+
+Renku ``0.31.0`` introduces the compute resource control (CRC) service and supporting or related changes in 
```

Comment:
Can we have a clear statement here about what this is for and what it enables?

Reply:
I have just proposed one.